docs/website/.gitignore

```diff
@@ -0,0 +1,26 @@
+# Dependencies
+/node_modules
```

Comment:
```suggestion
/node_modules
package-lock.json
```

This prevents a certain honking goose from making stupid pushes from their npm based docs builds... 🙃

Reply:
😏